test/images/agnhost/nettest/nettest.go

```diff
@@ -52,13 +54,30 @@ import (
 )
 
 var (
-	port          = flag.Int("port", 8080, "Port number to serve at.")
-	peerCount     = flag.Int("peers", 8, "Must find at least this many peers for the test to pass.")
-	service       = flag.String("service", "nettest", "Service to find other network test pods in.")
-	namespace     = flag.String("namespace", "default", "Namespace of this pod. TODO: kubernetes should make this discoverable.")
-	delayShutdown = flag.Int("delay-shutdown", 0, "Number of seconds to delay shutdown when receiving SIGTERM.")
+	port          int
+	peerCount     int
+	service       string
+	namespace     string
+	delayShutdown int
 )
 
+// CmdNettest is used by agnhost Cobra.
+var CmdNettest = &cobra.Command{
+	Use:   "nettest",
+	Short: "TBA",
+	Long:  "TBA",
+	Args:  cobra.MaximumNArgs(5),
```

Comment:
5 args? What are these for? It seems like 5 corresponds to the variables, but those are flags, not args. Did I overlook something?

Reply:
Done